tests/functional/upgrade/test_configuration.py

```diff
@@ -147,3 +148,75 @@ def test_update_strategy_config_change(
     )
     results = ds_obj.get_update_strategy()
     assert str(expected_value) == str(results["rollingUpdate"]["maxUnavailable"])
+
+
+@pytest.mark.parametrize(
+    argnames=["daemonset", "value_to_set"],
+    argvalues=[
+        pytest.param(
+            "csi-rbdplugin",
+            0,
+            marks=[pytest.mark.tier4a, pytest.mark.polarion_id("OCS-6517")],
+        ),
+        pytest.param(
+            "csi-rbdplugin",
+            "0%",
+            marks=[pytest.mark.tier4a, pytest.mark.polarion_id("OCS-6518")],
+        ),
+        pytest.param(
+            "csi-cephfsplugin",
+            0,
+            marks=[pytest.mark.tier4a, pytest.mark.polarion_id("OCS-6517")],
+        ),
+        pytest.param(
+            "csi-cephfsplugin",
+            "0%",
+            marks=[pytest.mark.tier4a, pytest.mark.polarion_id("OCS-6518")],
+        ),
+    ],
+)
+@brown_squad
+@runs_on_provider
+def test_max_unavailable_zero_update_strategy_config(
+    daemonset, value_to_set, rook_operator_configmap_cleanup
+):
+    """
+    Test that tested value added to configmap rook-ceph-operator-config is
+    updated with zero or zero percent value and generate the respective error
+    in rook-ceph-operator log
+    """
+    if daemonset == "csi-rbdplugin":
+        parameter_name = "CSI_RBD_PLUGIN_UPDATE_STRATEGY_MAX_UNAVAILABLE"
+    elif daemonset == "csi-cephfsplugin":
+        parameter_name = "CSI_CEPHFS_PLUGIN_UPDATE_STRATEGY_MAX_UNAVAILABLE"
+
+    config_map_patch = f'\'{{"data": {{"{parameter_name}": "{value_to_set}"}}}}\''
```

Comment:
For setting update strategy you can use function set_update_strategy (https://github.com/red-hat-storage/ocs-ci/blob/eedd97aeafd5dfb68682d2616b1b11da17e9bf59/ocs_ci/ocs/ocs_upgrade.py#L1020)